feature-libs/organization/administration/components/shared/detail/toggle-status-action/toggle-status.component.spec.ts

```diff
@@ -24,6 +24,8 @@ class MockMessageService {
 
 class MockItemService {
   current$ = of();
+  isInEditMode$: Observable<boolean> = new BehaviorSubject<boolean>(false);
```

Comment:
maybe we should test if this behavior is correct?

Reply:
yes indeed